### PR TITLE
Draft: PageTurnLayout becomes ViewPager with draw 

### PR DESCRIPTION
**Pull request by cesine**
This branch keeps the draw function in the ViewPager (former PageTurnLayout) which means it might be more successful than #6 in getting the curl animation to become visible. On the other hand, it has to use it's own page turn detection logic (maybe built on the onPageScrolled rather on than onTouch) which was the part that   never got triggered on a Nexus 10 in a case without pull #3

### CODE REVIEW

File: library/src/com/briangriffey/notebook/PageTurnViewPager.java [hunk 1]
```diff
@@ -182,8 +182,7 @@ public void run() {
 		return true;
 	}
 
-	@Override
-	public void draw(Canvas canvas) {
+	public void drawDeprecated(Canvas canvas) {
```
Comment (cesine) [inline library/src/com/briangriffey/notebook/PageTurnViewPager.java:186]: These methods are "off" resulting in teh view pager's default slide transition.

**Comment by cesine** [comment]
this branch has the least code modifications but doesnt have any page flip or curl or book-like visuals until someone who konws more about drawing than me figures it out.

**Comment by briangriffey** [comment]
Please only open a pull request when you are ready for a pull.

**Comment by cesine** [comment]
Sorry about that! How do you do a code review on github without a PR? i suppose i could do the PR between my own branches if you prefer

**Pull request closed without merging**